core/common/src/DateTimeUnit.kt

```diff
@@ -169,8 +169,8 @@ public sealed class DateTimeUnit {
      *
      * The reason lies in time zone transitions, because of which some days can be 23 or 25 hours.
      * For example, we say that exactly a whole day has passed between `2019-10-27T02:59` and `2019-10-28T02:59`
-     * in Berlin, despite the fact that the clocks were turned back one hour, so there are, in fact, 25 hours
-     * between the two date-times.
+     * in Berlin, even though the clocks were turned back one hour, so there are, in fact, 25 hours
+     * between the two date/times.
```

Comment:
The problem I was trying to solve here is the inconsistency between `date/time` and `date-time` in changed files in the original PR #367. Now, I've replaced all occurrences with `datetime`, everywhere except for test files

Reply:
Is `datetime` suitable both as the noun and as the adjective? The existence of a separate adjective https://en.wiktionary.org/wiki/date-time#English leaves me with a doubt.

Reply:
Yes, it's very common for nouns to act as adjectives, modifying other nouns
If we take this specific spelling for the [data type](https://en.wiktionary.org/wiki/datetime), it would be great to keep it consistent throughout the doc, especially since it's a key concept for the library